spec/lib/models/task-definition-spec.js

```diff
@@ -4,9 +4,18 @@
 'use strict';
 
 var base = require('./base-spec');
-
+var  sandbox = sinon.sandbox.create();
```

Comment:
Would be nice to have a newline here, it just makes it a little easier to see when module global variables are being declared.